src/Core/Core/Execution/Instrumentation/ApolloTracingDiagnosticObserver.cs

```diff
@@ -115,5 +150,14 @@ private static long GetStartTimestamp()
             return Convert.ToInt64(Activity.Current.Tags
                 .First(t => t.Key == _startTimestampKey).Value);
         }
+
+        private bool IsEnabled(object context)
```

Comment:
So, the outer IsEnabled is now completely obsolete?
To have it here in the Apollo Tracing Observer seems even to make more sense, since this is only related to this specific observer.